drivers/base/bootvid/i386/pc/vga.c

```diff
@@ -197,81 +188,8 @@ DisplayCharacter(
     }
 }
 
-static VOID
-NTAPI
-SetPaletteEntryRGB(IN ULONG Id,
-                   IN ULONG Rgb)
-{
-    PCHAR Colors = (PCHAR)&Rgb;
-
-    /* Set the palette index */
-    __outpb(VGA_BASE_IO_PORT + DAC_ADDRESS_WRITE_PORT, (UCHAR)Id);
-
-    /* Set RGB colors */
-    __outpb(VGA_BASE_IO_PORT + DAC_DATA_REG_PORT, Colors[2] >> 2);
-    __outpb(VGA_BASE_IO_PORT + DAC_DATA_REG_PORT, Colors[1] >> 2);
-    __outpb(VGA_BASE_IO_PORT + DAC_DATA_REG_PORT, Colors[0] >> 2);
-}
-
 VOID
 NTAPI
-InitPaletteWithTable(
-    _In_ PULONG Table,
-    _In_ ULONG Count)
-{
-    ULONG i;
-    PULONG Entry = Table;
-
-    /* Loop every entry */
-    for (i = 0; i < Count; i++, Entry++)
-    {
-        /* Set the entry */
-        SetPaletteEntryRGB(i, *Entry);
-    }
-}
-
-static VOID
-NTAPI
-SetPaletteEntry(IN ULONG Id,
-                IN ULONG PaletteEntry)
```

Comment:
Function similar to https://git.reactos.org/?p=reactos.git;a=blob;f=subsystems/mvdm/ntvdm/bios/vidbios.c;hb=592bafd60a33b82e62174f124ce80b2e0374a03a#l2242